src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
```

Comment:
Consider adding a fallback font family for better cross-browser compatibility. For instance, if your design uses a sans-serif font, you can specify `font-family: 'YourFont', sans-serif;`.